script.js

```diff
@@ -0,0 +1,17 @@
+const leftButton = document.querySelector(".left-btn");
+const rightButton = document.querySelector(".right-btn");
+const carousel = document.querySelector(".carousel");
+
+const scroll = (direction) => {
+  let width =
+    Math.floor(document.querySelector(".carousel-item").scrollWidth) + 16;
```

Comment:
Can we not do this `querySelector` outside `scroll` method?
```js
const carouselItem = document.querySelector(".carousel-item");
....

let width = Math.floor(carouselItem.scrollWidth) + 16;
...
```

Reply:
extract `16` as a constant.

Reply:
> Can we not do this `querySelector` outside `scroll` method?
> 
> ```js
> const carouselItem = document.querySelector(".carousel-item");
> ....
> 
> let width = Math.floor(carouselItem.scrollWidth) + 16;
> ...
> ```

Yes, we can.